### ISSUE TITLE
[BUG] Graph not showing up with remote access

### ISSUE THREAD

**Issue by smoochy** [jgulick48/NiceHashProfitabilityCalculator#37]
I used your tool a few days ago, when I was still at home and I could see the graph in the Rig Stats and Card Stats normally.

Now I am on vacation, want to occasionally check how it is going.

For remote access I used TeamViwer and also Google Chrome Remote Access.
With both applications I get the same result.

I start a Graphs view. The windows stays empty, but when I mouse over the window, I see the little info pop ups (the same I asked on Reddit to show information about the algorithm)

For better understanding, here a screenshot showing what I mean:

![bug](https://user-images.githubusercontent.com/34371932/35216641-91bd47e0-ff68-11e7-9623-6f012992045a.png)

Not sure where exactly this is coming from, as it was working fine when sitting directly in front of the computer.

**Comment by jgulick48** [comment]
It looks like this may be an issue with TeamViewer and not refreshing the screen properly for some reason.  Since the viewer can be run on any computer I would recommend viewing it on your local machine rather than remoting in to view it there.

**Comment by smoochy** [comment]
At first I also thought it would be caused by TeamViewer. This is why I tested Google Remode Desktop.

But there is the same behaviour. Hence I think this might also appear with different remote desktop tools and might affect more people.

I am currently using a Mac here on my vacation. I could open up a VPN connection to my home network, but can not run your application. So I am forced to use a remote desktop solution.

> Since the viewer can be run on any computer

Does this mean with a virtual machine running on my Mac I can use the viewer, even when I am abroad?

**Comment by jgulick48** [comment]
That is correct, if you can run it in a VM on your mac you can run it there.  It logs everything to a central database that I'm hosting for the time being and thus the viewer works remotely.
